### PR TITLE
Update rekor instance URL and fulcio key algorithm

### PR DESCRIPTION
**Pull request by mayaCostantini**
#### Summary

- Update the public instance URL for rekor
- Update the DES algorithm to DES3 as the former is [not included out of the box from OpenSSL v3.0](https://www.openssl.org/docs/man3.1/man7/migration_guide.html)

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: src/10-sign-container.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ export SIGSTORE_ROOT_FILE="$HOME/fulcio-root.pem"
 Download the Rekor public key:
 
 ```bash
-wget -O publicKey.pem https://rekor.ozz-sigstore-instance.dev/api/v1/log/publicKey
+wget -O publicKey.pem https://rekor.sigstore.dev/api/v1/log/publicKey
```
Comment (JAORMX) [inline src/10-sign-container.md]: Hey! Thanks a lot for fixing things. The intent is for folks to download the public key from the rekor instance they're deploying, so "https://rekor.example.com" would be more appropriate. Thanks for pointing out this typo!

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: src/10-sign-container.md (hunk 1 above)
Comment (mayaCostantini) [reply src/10-sign-container.md]: Right, thanks! Changed in https://github.com/lukehinds/sigstore-the-hard-way/pull/32/commits/0c20f6e6991d859d59cef2754008271d62324e71

**Review by JAORMX — APPROVED** [review]

**Comment by JAORMX** [comment]
Thanks for the fixes!

**Comment by JAORMX** [comment]
The lint issues seem unrelated. @lukehinds could you take a look?

**Pull request merged**